app/views/admin/stats/top_balances/index.html.slim

```diff
@@ -0,0 +1,17 @@
+div.panel-body
+  div.adv-table
+    table.display.table.table-bordered.table-striped.dynamic-table id="dynamic-table"
+      tr
+        th Account Type
```

Comment:
Might make more sense to swap Account Type and Account ID. For example, against staging data

<img width="1792" alt="screen shot 2019-02-05 at 6 20 56 pm" src="https://user-images.githubusercontent.com/5459225/52313087-cb941180-2972-11e9-9ce4-6cce57c280ba.png">